src/flamegraph.cpp

```diff
@@ -447,6 +448,16 @@ bool FlameGraph::eventFilter(QObject* object, QEvent* event)
         updateTooltip();
     } else if (event->type() == QEvent::Hide) {
         setData(nullptr);
+    } else if (event->type() == QEvent::ContextMenu) {
+        QMenu contextMenu;
+        auto *viewCallerCallee = contextMenu.addAction(tr("View Caller/Callee"));
+
+        QContextMenuEvent *contextEvent = static_cast<QContextMenuEvent*>(event);
+        auto item = static_cast<FrameGraphicsItem*>(m_view->itemAt(m_view->mapFromGlobal(contextEvent->globalPos())));
```

Comment:
actually, when no item is found, we shouldn't even offer the context menu, right? so please change this to check for an item first before creating the menu and action